### PR TITLE
API Assignment- 2

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: APIAssignment/pom.xml [hunk 1]
```diff
@@ -29,8 +29,35 @@
...
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.12</version>
+            <scope>test</scope>
+        </dependency>
 
 
-    </dependencies>
```
Comment (prashantkawa) [inline APIAssignment/pom.xml]: remove extra spaces

File: APIAssignment/src/test/java/APIAssignments/ProductLists.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.apache.log4j.PropertyConfigurator;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.logging.Logger;
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/ProductLists.java]: Use Logger from apache.Log4j library

File: APIAssignment/src/test/java/APIAssignments/ProductLists.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    public ProductLists() {
+        try {
+            url = ConfigReader.getUrl();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static Logger logger = Logger.getLogger("ProductLists.class");
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/ProductLists.java]: Declare logger as final

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: APIAssignment/pom.xml (hunk 1 above)
Comment (anushreeSonak) [reply APIAssignment/pom.xml]: done !!

File: APIAssignment/src/test/java/APIAssignments/ProductLists.java (hunk 2 above)
Comment (anushreeSonak) [reply APIAssignment/src/test/java/APIAssignments/ProductLists.java]: done !!

File: APIAssignment/src/test/java/APIAssignments/ProductLists.java (hunk 3 above)
Comment (anushreeSonak) [reply APIAssignment/src/test/java/APIAssignments/ProductLists.java]: done !!

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**